assets/training/finetune_acft_image/environments/acft_image_medimageinsight_embedding/context/requirements.txt

```diff
@@ -0,0 +1,28 @@
+azureml-acft-common-components==0.0.61
+azureml-acft-contrib-hf-nlp==0.0.61
+opencv-python==4.11.0.86
+pyyaml==6.0.2
+json_tricks==3.17.3
+yacs==0.1.8
+scikit-learn==1.5.2
+pandas==2.0.3
+timm==0.4.12
+numpy==1.22.2
+einops==0.8.1
+fvcore==0.1.5.post20221221
+transformers==4.16.2
+sentencepiece==0.2.0
+ftfy==6.3.1
+regex==2024.11.6
+vision-datasets==0.2.7
+tenacity==9.0.0
+requests==2.32.3
+azure-storage-blob==12.11.0
+deepspeed==0.6.3
+GPUtil==1.4.0
+mup==1.0.0
+mlflow==2.20.2
+azureml-mlflow==1.59.0.post1
+torchvision==0.14.1
+torch==1.13.1 
```

Comment:
vulnerabilities":[{"aliases":["CVE-2024-31583"],"details":"Pytorch before version v2.2.0 was discovered to contain a use-after-free vulnerability in torch/csrc/jit/mobile/interpreter.cpp.","fixed_in":["2.2.0"],"id":"GHSA-pg7h-5qx3-wjr3","link":"https://osv.dev/vulnerability/GHSA-pg7h-5qx3-wjr3","source":"osv","summary":null,"withdrawn":null}]}
https://[pypi.org/pypi/torch/1.13.1/json](https://pypi.org/pypi/torch/1.13.1/json) unless you want a middle of the night call to your DRI you should consider to update your dependencies